model/client4.go

```diff
@@ -1146,6 +1146,16 @@ func (c *Client4) RevokeAllSessions(userId string) (bool, *Response) {
 	return CheckStatusOK(r), BuildResponse(r)
 }
 
+// RevokeAllSessions revokes all sessions for the provided user id string.
+func (c *Client4) RevokeSessionsFromAllUsers(userID string) (bool, *Response) {
```

Comment:
With the changes I suggested in the `api4` file above, this no longer needs a UserID parameter. The doc comment above should be updated too.